src/backend/tests/integration/backward_compatibility/test_starter_projects.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from langflow.schema.message import Message
+from tests.api_keys import get_openai_api_key
+from tests.integration.utils import download_flow_from_github, run_json_flow
+
+
+@pytest.mark.asyncio
+@pytest.mark.api_key_required
+async def test_1_0_15_basic_prompting():
```

Comment:
very cool, I like this compromise of testing actual json and not storing a full folder of flows in our repo

Reply:
Very cool!